### DISCUSSION TITLE
[Projects Beta] Would be helpful to have the same filtering options as Github issues.

### DISCUSSION THREAD

**Discussion by 0xabrar** [community/community#11792] [Projects and Issues]
The current filtering on project views feels super limited. Sure, I can filter by keywords or by field, but I feel like I should be able to filter out issues the same way normal issues are filtered.

As an example, for one of my project views, if I only wanted to display recently closed issues, I could add:

`closed:>2022-02-18`

But that's not an option.

**Comment by NichUK** [comment]
I completely agree. We also need to be able to use OR.  When creating a view to see things with status == Todo, I also want to include things with status == none, so I can make sure nothing gets lost or forgotten, but you don't seem to be able to do that. Everything is an AND.
